src/lib/reducer.js

```diff
@@ -62,7 +62,7 @@ function reduceAddSubExpression(node, precision) {
     isValueType(node.left.type) &&
     node.left.value === 0 &&
     node.operator === "-" &&
-    node.right.type !== "Function"
+    !containCSSFunction(node.right)
```

Comment:
Let's rename `isFunction` or `isCSSFunction`

Reply:
But it is a recursive process, isn't "containXXX" better than "isXXX" here?

Reply:
I'm fine with both! Already changed according to your advice.